modules/migration-configuring-proxy-for-dvm.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+// * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
+
+[id='migration-configuring-proxy-for-dvm_{context}']
+= Configuring a proxy for direct volume migration
+
+If you are performing direct volume migration from a source cluster behind a proxy, you must configure an Stunnel proxy in the `MigrationController` custom resource (CR). Stunnel creates a transparent tunnel between the source and target clusters for the TCP connection without changing the certificates.
```

Comment:
an Stunnel -> a Stunnel

Reply:
No. The general rule is that the form of the indefinite article depends on pronunciation. Stunnel is pronounced "Ess tunnel".

Reply:
Ah, I wasn't aware it had that pronunciation. That's a weird spelling for it to be sounded out that way.

Reply:
It would have been clearer if they had hyphenated the name (S-tunnel) but, unfortunately, we have no control over how other companies spell their product names. :-)